kf_lib_data_ingest/common/stage.py

```diff
@@ -153,6 +166,16 @@ def run(self, *args, **kwargs):
         # Write output of stage to disk
         self.write_output(output)
 
+        # Write data for accounting to disk
         concept_discovery_dict = self._postrun_concept_discovery(output)
-
-        return output, concept_discovery_dict
+        self.concept_discovery_dict = concept_discovery_dict
```

Comment:
need this `concept_discovery_dict`? why not `self.concept_discovery_dict = ...`

Reply:
👌